server/static/js/notebook.js

```diff
@@ -0,0 +1,30 @@
+function initalizeNotebookMarkdown() {
+    // Customize the markdown function
+    var mdRenderer = window.markdownit({
+        html: true
```

Comment:
The authors of markdownit warn against enabling html: True (for XSS reasons) - but recommend using plugins like markdownItSanitizer (which only allows _very_ basic markdown and neutralizes everything else)